### DISCUSSION TITLE
Deploy Flags Documentation

### DISCUSSION THREAD

**Discussion by jimmyddalton** [wpengine/github-action-wpe-site-deploy#85] [Q&A]
As I'm reading through the documentation for the github action I'm looking to take advantage of the Flags but documentation seems a little thin. Is there a list of what flags are available and what they do? For example, what does --inplace do? I've also seen use of --backup.

My main challenge at the moment is when I delete files from my repo on my local development environment they are not being deleted when I push to production. I end up having to SSH into my prod environment and manually delete files. While this is the main issue I'm trying to solve, if there are other flags that I could benefit from I'd love to see the docs.

Thanks.

**Comment by kmaling** [comment]
Did you ever get anywhere with this? I'm trying to build a basic dev-staging-production workflow, but using this action, none of the files that I delete in my local environment get removed from the WPE server.

For example, if I add a file in my local environment and push that to GitHub, it deploys that file to the WPE server. All good. But, if I then delete that file in my local environment, and push the change to GitHub, while it's removed from the GitHub repo, after the deploy action completes, the file is still on the WPE server?

**Reply by jimmyddalton** [reply]
Unfortunately, I haven't heard anything from WP Engine support. I have a feeling there might be something with the exclude file.

**Reply by jimmyddalton** [reply]
So I think I have been able to get this to work.
- I used the example from the documentation in GitHub: https://github.com/wpengine/github-action-wpe-site-deploy
- FLAGS: -azvr --inplace --delete --exclude=".*"  --exclude=wp-content/mu-plugins/local-plugin --exclude-from=exclude.txt
- Added the exclude file they reference: https://github.com/wpengine/site-deploy/blob/main/exclude.txt
- My theme has composer dependencies so I had to add a shell script to reinstall the dependencies.

It took some experimenting on a staging slot but I think I've got it working.
